packages/react-composites/src/composites/MeetingComposite/adapter/AzureCommunicationMeetingAdapter.ts

```diff
@@ -397,8 +408,7 @@ export type AzureCommunicationMeetingAdapterArgs = {
   userId: CommunicationUserIdentifier;
   displayName: string;
   credential: CommunicationTokenCredential;
-  chatThreadId: string;
-  callLocator: TeamsMeetingLinkLocator | GroupCallLocator;
+  meetingArgs: CallAndChatMeetingArgs | TeamsMeetingLinkLocator;
```

Comment:
Could we call this `meetingLocator` to be consistent with `callLocator` from `CallComposite` and because "args" is too generic.

Reply:
Yeah I don't like args, my worry with locator is chat avoids the term locator -- but I like it more than args so will update

Reply:
updated